<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>755</width>
    <height>367</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Capillary Tube Parametric Study</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_6">
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Parameters</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_14">
          <item>
           <widget class="QCheckBox" name="capillary_length_check">
            <property name="text">
             <string>Capillary Length</string>
            </property>
           </widget>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_14">
            <item>
             <widget class="QLineEdit" name="capillary_length">
              <property name="enabled">
               <bool>false</bool>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QComboBox" name="capillary_length_unit">
              <property name="enabled">
               <bool>false</bool>
              </property>
              <item>
               <property name="text">
                <string>m</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>cm</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>mm</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>ft</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>in</string>
               </property>
              </item>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_17">
          <item>
           <widget class="QCheckBox" name="capillary_D_check">
            <property name="text">
             <string>Capillary Diameter</string>
            </property>
           </widget>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_17">
            <item>
             <widget class="QLineEdit" name="capillary_D">
              <property name="enabled">
               <bool>false</bool>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QComboBox" name="capillary_D_unit">
              <property name="enabled">
               <bool>false</bool>
              </property>
              <item>
               <property name="text">
                <string>m</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>cm</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>mm</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>ft</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>in</string>
               </property>
              </item>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_18">
          <item>
           <widget class="QCheckBox" name="Capillary_entrance_D_check">
            <property name="text">
             <string>Capillary Entrance Diameter</string>
            </property>
           </widget>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_18">
            <item>
             <widget class="QLineEdit" name="Capillary_entrance_D">
              <property name="enabled">
               <bool>false</bool>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QComboBox" name="Capillary_entrance_D_unit">
              <property name="enabled">
               <bool>false</bool>
              </property>
              <item>
               <property name="text">
                <string>m</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>cm</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>mm</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>ft</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>in</string>
               </property>
              </item>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </item>
        <item>
         <spacer name="verticalSpacer_2">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_5">
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_19">
          <item>
           <widget class="QCheckBox" name="Capillary_N_tubes_check">
            <property name="text">
             <string>Number of Parallel Tubes</string>
            </property>
           </widget>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_19">
            <item>
             <widget class="QLineEdit" name="Capillary_N_tubes">
              <property name="enabled">
               <bool>false</bool>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_20">
          <item>
           <widget class="QCheckBox" name="Capillary_DP_tolerance_check">
            <property name="text">
             <string>Pressure Drop Tolerance</string>
            </property>
           </widget>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_20">
            <item>
             <widget class="QLineEdit" name="Capillary_DP_tolerance">
              <property name="enabled">
               <bool>false</bool>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QComboBox" name="Capillary_DP_tolerance_unit">
              <property name="enabled">
               <bool>false</bool>
              </property>
              <item>
               <property name="text">
                <string>Pa</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>kPa</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>MPa</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>bar</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>atm</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>psi</string>
               </property>
              </item>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_21">
          <item>
           <widget class="QCheckBox" name="Capillary_DT_check">
            <property name="text">
             <string>Discretzation Temperature for Two-phase</string>
            </property>
           </widget>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_21">
            <item>
             <widget class="QLineEdit" name="Capillary_DT">
              <property name="enabled">
               <bool>false</bool>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QComboBox" name="Capillary_DT_unit">
              <property name="enabled">
               <bool>false</bool>
              </property>
              <item>
               <property name="text">
                <string>C</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>F</string>
               </property>
              </item>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </item>
        <item>
         <spacer name="verticalSpacer_3">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_2">
     <property name="title">
      <string>Hint</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <widget class="QLabel" name="label">
        <property name="text">
         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Enter comma separated values for parameters. for example:&lt;br&gt;3, 5, 7 ,11&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="cancel_button">
       <property name="text">
        <string>Cancel</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="ok_button">
       <property name="text">
        <string>Ok</string>
       </property>
       <property name="default">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_3">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
